app/controllers/api/v1/movies/movies_controller.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
```

Comment:
Due to the scope of changes required, i suggest to move `movies_controller.rb` out of `movies` folder. Since I don't see necessary of that Movie module

Reply:
Ready